src/App.js

```diff
@@ -1,10 +1,54 @@
+import styled, { createGlobalStyle } from 'styled-components';
+
+import Todo from './components/Todo';
 
 function App() {
   return (
-    <div>
-      <h1>15기 화이팅~</h1>
-    </div>
+    <Wrapper>
+      <GlobalStyle />
+      <Main>
+        <MainTitle>할 일 목록</MainTitle>
+        <Todo />
+      </Main>
+    </Wrapper>
   );
 }
 
+const Wrapper = styled.div`
+  width: 100%;
+`;
+
+const GlobalStyle = createGlobalStyle`
+  *{  
+    box-sizing: border-box;
+    font-family: 'Noto Sans KR', sans-serif;
+  }
+  html {
+    font-size : 16px;
+  }
+  body {
+    background-color : #F6FFDB;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 100%;
+    width: 99%;
+  }
+`;
+
+const Main = styled.div`
+  width: 27rem;
+  height: 50rem;
+  background-color: #fff;
+  border-radius: 2rem;
+  box-shadow: rgba(0, 0, 0, 0.2) 0 0 2rem;
+  padding: 1.5rem 2rem 1.5rem 2rem;
```

Comment:
상하/좌우 패딩값이 같아서 값을 아래처럼 줘도 될 것 같아요!

```suggestion
  padding: 1.5rem 2rem;
```

Reply:
아직 익숙하지 않아서 4개 다 사용했는데, 알려주셔서 감사합니다!!